static/js/pages/elections.js

```diff
@@ -83,22 +83,17 @@ function makeCommitteeColumn(opts, factory) {
   return _.extend({}, {
     orderSequence: ['desc', 'asc'],
     render: tables.buildTotalLink(['receipts'], function(data, type, row, meta) {
+      row.cycle = context.election.cycle;
       var column = meta.settings.aoColumns[meta.col].data;
       return _.extend({
-        committee_id: (context.candidates[row.candidate_id] || {}).committee_ids,
-        cycle: context.election.cycle
+        committee_id: (context.candidates[row.candidate_id] || {}).committee_ids
```

Comment:
Identifier 'committee_id' is not in camel case.<br>Identifier 'candidate_id' is not in camel case.<br>Identifier 'committee_ids' is not in camel case.